src/domains/printable.ml

```diff
@@ -530,7 +560,7 @@ end
 
 module Strings =
 struct
-  type t = string [@@deriving eq, to_yojson]
+  type t = string [@@deriving eq, yojson]
```

Comment:
Why does this need to also derive `of_yojson`?

Reply:
It doesn't need to. This was probably accidental.